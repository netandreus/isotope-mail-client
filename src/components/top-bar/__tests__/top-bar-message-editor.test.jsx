import React from 'react';
import {shallow} from 'enzyme/build/index';
import TopBarMessageEditor from '../top-bar-message-editor';

describe('TopBarMessageEditor component test suite', () => {
  test('Snapshot render, should render top bar message editor', () => {
    // Given
    const props = {collapsed: true, title: 'sideshow bob', sideBarToggle: jest.fn()};
    // When
    const topBarMessageEditor = shallow(<TopBarMessageEditor {...props}/>);
    // Then
    expect(topBarMessageEditor).toMatchSnapshot();
  });
});
