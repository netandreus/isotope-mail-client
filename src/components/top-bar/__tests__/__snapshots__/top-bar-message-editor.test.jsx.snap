// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBarMessageEditor component test suite Snapshot render, should render top bar message editor 1`] = `
<div
  className="mdc-top-app-bar__row"
>
  <section
    className="mdc-top-app-bar__section mdc-top-app-bar__section--align-start"
  >
    <Translate(ButtonCollapse)
      collapsed={true}
      sideBarToggle={[MockFunction]}
    />
    <span
      className="mdc-top-app-bar__title"
    >
      sideshow bob
    </span>
  </section>
</div>
`;
